change/@microsoft-teams-js-6d28f626-b164-4d85-ad16-098e9b7491dd.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "Added search capability to use global search box in the current app in Outlook",
```

Comment:
the first "search" should be surrounded by backticks (`search`) because it is a reference to something in code.